code/modules/antagonists/revenant/revenant_blight.dm

```diff
@@ -25,42 +25,42 @@
 	..()
 
 
-/datum/disease/revblight/stage_act()
+/datum/disease/revblight/stage_act(delta_time, times_fired)
 	. = ..()
 	if(!.)
 		return
 
 	if(!finalstage)
-		if(affected_mob.body_position == LYING_DOWN && prob(stage * 6))
+		if(affected_mob.body_position == LYING_DOWN && DT_PROB(3 * stage, delta_time))
 			cure()
 			return FALSE
-		if(prob(stage * 3))
+		if(DT_PROB(1.5 * stage, delta_time))
 			to_chat(affected_mob, "<span class='revennotice'>You suddenly feel [pick("sick and tired", "disoriented", "tired and confused", "nauseated", "faint", "dizzy")]...</span>")
 			affected_mob.add_confusion(8)
 			affected_mob.adjustStaminaLoss(20, FALSE)
 			new /obj/effect/temp_visual/revenant(affected_mob.loc)
 		if(stagedamage < stage)
 			stagedamage++
-			affected_mob.adjustToxLoss(stage * 2, FALSE) //should, normally, do about 30 toxin damage.
+			affected_mob.adjustToxLoss(1 * stage * delta_time, FALSE) //should, normally, do about 30 toxin damage.
```

Comment:
If you turn up the delta_time it will end up becoming more and more likely to just delete itself on the first tick and not actually do anything. The only way I can see to fix that would be to move that bit until after it handles the effects, which would be it's own behavior change.

Reply:
Done